pom.xml

```diff
@@ -159,6 +159,11 @@
             <version>2.0.8</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
The dependency has to be optional, and you should check for its presence.

Reply:
I updated the dependecy management (in Util) a bit, pls check it out

Reply:
Perfect!